doc/media/evolution-openage-gource.md

```diff
@@ -0,0 +1,18 @@
+# Reproducing the gource 'evolution video'
+
+Download, compile & install git, gource, ffmpeg + FLOSS video cutting tool.
+
+1. Set git directory to 'Master' branch
+2. Open terminal/command line and create a new folder for the video
+3. Copy the logo in *.png-format into the new folder (svg->png with Inkscape)
+4. Navigate to the new folder
+5. Use the following command
+
+'gource --camera-mode overview --title https://openage.sft.mx --seconds-per-day 0.05 --auto-skip-seconds 0.1 --max-file-lag 0.1 --file-idle-time 0 --background 555544 --logo *.png --font-size 15 --date-format "%B %Y" --hide filenames, mouse -1920x1080 -o <outputfile-name.ppm> <../path to git directory>'
```

Comment:
Code/commands should be surrounded by  

\`\`\`
gource -xyz bla
\`\`\`

result:

```
gource -xyz bla
```